about.html

```diff
@@ -0,0 +1,138 @@
+          <h1 class="abt-h1">
+            2<sup>nd</sup> Global Submit on Applied Science, Engineering &<br>
+            Technology
+          </h1>
+        </div>
+        <div class="about-conference">
+          <p>
+            Applied-Science-2023 provides a wonderful platform in bringing
+            together leading scientists, research scholars, research students,
+            practitioners, engineers, academic professionals and policy makers
+            with highly cited plenary lectures, keynote lectures and invited
+            talks to share their latest innovations in the fields of Applied
+            Science, Engineering and technology.
+          </p>
+          <p>
+            In addition, variety of Young research Forums, poster forum along
+            with workshops, special sessions, contributed papers, video and oral
+            poster presentations covering all frontier topics on latest research
+            progress and innovations would be interested in audience. We make
+            sure to support and motivate Young Researchers and Students “3 Best
+            Young Researcher Award” and “3 Best Poster Award” at
+            Applied-Science-2023 Platform.
+          </p>
+        </div>
+        <p>
+          Please contact us via Email for any further questions about the
+          Science Conference 2023
+        </p>
+        <a href="">gsaset2023@thescientistt.com</a>
+      </div>
+    </section>
+
+    <section class="about-the-logo">
+      <div class="about-logo-content">
+        <h3>Global Submit on Applied Science, Engineering & Technology logo</h3>
+        <p>The logo of the Global submit was decided through the logo competition form 8th June to 7th July. After online votiong and election commitee's judging process, the logo from Nanjal Gumbson won the price.<br>After consulting Nanjal Gumbson, we optimized the prize winning logo for this years as seen below.</p>
+        <div class="logo-holder">
+          <img src="./img/logo4.png" alt="Company Logo">
+        </div>
+      </div>
+    </section>
+
+    <section class="past-submit">
+      <div class="past-submit-container container">
+        <h2>See the past GSASET Submits</h2>
+        <p>Take a look at the past two submits</p>
+        <div class="submits">
+          <div class="submit1">GSASET Submit<br>2022</div>
+          <div class="submit2">GSASET Submit<br>2022</div>
+      </div>
+    </section>
+
+    <section class="partners">
+      <div class="partners-content container">
+        <h2 class="partners-p">Partners</h2>
+        <div class="partners-holder">
+          <img class="partners-logo" src="./img/mozila.png" alt="mozilla icon" />
+          <img class="partners-logo" src="./img/google.png" alt="google icon" />
+          <img class="partners-logo" src="./img/airbnb (1).png" alt="airbnb icon" />
+          <img class="partners-logo" src="./img/naver.png" alt="naver icon" />
+        </div>
+      </div>
+    </section>
```

Comment:
# Desktop Size and Mobile Size of About Page: OPTIONAL
- Please these pages don't have a partners section. Open this 👉 [ Link to learn more](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) about the design
- Also make sure the last section of the footer in desktop size is black in color.
- 
-**Your design**:
![Capture](https://user-images.githubusercontent.com/63972670/197008303-541a4af2-ee87-43b2-b93b-7dd731dfe8e7.JPG)
- o
- **See the Original design**:
![Capture](https://user-images.githubusercontent.com/63972670/197008590-38e35ce1-2a44-445d-97b4-8c091b30536e.JPG)